config/initializers/simple_form_bootstrap.rb

```diff
@@ -0,0 +1,440 @@
+# frozen_string_literal: true
```

Comment:
I don't think adding this is necessary. To make the feature as simple and maintainable as possible, it should use existing UI patterns and follow the conventions of the rest of the codebase; I don't want to add a large amount of initializer code like this that I don't fully understand.

Reply:
Ok, I will remove Bootstrap UI.